src/utils/constants.ts

```diff
@@ -0,0 +1,102 @@
+export const DEPRECATED_GRAPH_URL =
+  'https://graph-node.avax.network/subgraphs/name/dasconnor/pangolindex';
+export const GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/dasconnor/pangolin-dex';
+export const RPC_URL = 'https://api.avax.network/ext/bc/C/rpc';
+
+export const PNG_ADDRESS = '0x60781C2586D68229fde47564546784ab3fACA982';
+export const WAVAX_ADDRESS = '0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7';
+export const WAVAX_PNG_ADDRESS = '0xd7538cABBf8605BdE1f4901B47B8D42c61DE0367';
+export const FACTORY_ADDRESS = '0xefa94DE7a4656D787667C749f7E1223D71E9FD88';
+export const TREASURY_VESTER_ADDRESS = '0x6747AC215dAFfeE03a42F49FebB6ab448E12acEe';
+export const COMMUNITY_TREASURY_ADDRESS = '0x650f5865541f6D68BdDFE977dB933C293EA72358';
+
+import {BigNumber} from '@ethersproject/bignumber';
+export const ONE_TOKEN = BigNumber.from('1000000000000000000');
+export const TOTAL_SUPPLY = ONE_TOKEN.mul(538000000);
+
+// https://github.com/pangolindex/interface/blob/master/src/state/stake/hooks.ts
+export const STAKING_ADDRESSES = [
+  '0x417c02150b9a31bcacb201d1d60967653384e1c6', // AVAX-ETH
```

Comment:
we have new addresses for the VSO token, you can get the address from my latest interface PR